repository/Seaside-Examples.package/WASessionProtectedCounter.class/class/initialize.st

```diff
@@ -2,4 +2,5 @@ initialization
 initialize
 	| app |
 	app := WAAdmin register: self asApplicationAt: self handlerName.
+	app configuration at: #trackingStrategy put: WACookieOnlySessionTrackingStrategy new.
```

Comment:
The tracking strategy should be the default one (WAQueryFieldHandlerTrackingStrategy)